### ISSUE TITLE
getChatAdministrator shenanigans

### ISSUE THREAD

**Issue by bengsparks** [StefanoBelli/xxtelebot#28]
I wrote the following code to detect whether a user is an admin or not.

```
bool admin::methods::is_admin(const tgbot::types::Message &msg, const tgbot::methods::Api &api) {
	const tgbot::types::Chat chat_type = api.getChat(std::to_string(msg.chat.id));
	if (chat_type.type == tgbot::types::ChatType::PRIVATE) {
		return false;
	}

	const auto& admins = api.getChatAdministrators(std::to_string(msg.chat.id));
	return std::find_if(admins.begin(), admins.end(), [&](const tgbot::types::ChatMember& member) {
		return member.user.id == msg.from->id;
	}) != admins.end();
}
```

When entering the command that calls this function in the telegram supergroup, the bot SIGABRTs with the following exception:
```
terminate called after throwing an instance of 'Json::LogicError'
  what():  Value is not convertible to Int.
```

The output from GET "https://api.telegram.org/botTOKEN/getChatAdministrators?chat_id=CHAT_ID" appears to be correct as well, but I won't post it unless absolutely necessary, as it contains some personal information.

Am I doing something wrong or is something the method bugged?

In case it is relevant, here is my compiler information
```
clang version 6.0.0 (tags/RELEASE_600/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```

**Comment by StefanoBelli** [comment]
Sorry for being late and for this issue. This is on my own, I'll close this issue when this gets fixed

**Comment by StefanoBelli** [comment]
Yup, I think it couldn't convert until_date from ChatMember. 

Error of mine.

It is an optional so wasn't in your response.

I'll fix tomorrow

**Comment by StefanoBelli** [comment]
Fixed (e1b10a3)

Closing issue.

Pull changes, I won't publish new release, for now use the git-master version. Thanks for reporting this bug! Don't hesitate to open new ones and seek for help!
